src/Microsoft.OpenApi.OData.Reader/Edm/ODataPathProvider.cs

```diff
@@ -360,6 +377,19 @@ private void CreateRefPath(ODataPath currentPath)
             newPath.Push(ODataRefSegment.Instance); // $ref
             AppendPath(newPath);
         }
+        /// <summary>
+        /// Create $count paths.
+        /// </summary>
+        /// <param name="currentPath">The current OData path.</param>
+        /// <param name="convertSettings">The settings for the current conversion.</param>
+        private void CreateCountPath(ODataPath currentPath, OpenApiConvertSettings convertSettings) {
+            if(currentPath == null) throw new ArgumentNullException(nameof(currentPath));
```

Comment:
make a better format

Reply:
for others

Reply:
I'm assuming that you're referring to the curly brace not being on the next line. Fixed.

Reply:
if(currentPath == null)
{
      throw new ArgumentNullException(nameof(currentPath));
 }
 
 and, maybe to use Error.ArgumentNull(...) method?